themes/default/tasks/rotate_comment_readlog.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl -w
+# This code is a part of Slash, and is released under the GPL.
+# Copyright 1997-2005 by Open Source Technology Group. See README
+# and COPYING for more information, or see http://slashcode.com/.
+# $Id$
+
+use strict;
+use utf8;
+use Slash;
+use Slash::Constants ':slashd';
+use vars qw( %task $me %redirects );
+
+$task{$me}{timespec} = '30 1 * * *';
```

Comment:
Yup. Completely arbitrary time but I don't think it really needs to be run any more often than daily.